opencood/models/point_pillar_where2comm.py

```diff
@@ -95,17 +95,16 @@ def forward(self, data_dict):
 
         # Compressor
         if self.compression:
+            # The ego feature is also compressed
             spatial_features_2d = self.naive_compressor(spatial_features_2d)
 
         if self.multi_scale:
+            # Bypass communication cost, communicate at high resolution, neither shrink nor compress
             fused_feature, communication_rates = self.fusion_net(batch_dict['spatial_features'],
                                                                  psm_single,
                                                                  record_len,
                                                                  pairwise_t_matrix,
                                                                  self.backbone)
-            # Down-sample feature to reduce memory
```

Comment:
We need to keep this. Because all the models in V2X-ViT paper shrinked the feature, which can highly influence the performance. We need to make the comparison fair and thus keep this option

Reply:
Oh, I forgot to mention that. Honestly, I think Where2comm has something confusing in its multi-scale implementation. You will find that it re-extracts the features in multi-scale implementation. It makes the order of **shrink**, **compress**, **fuse** somewhat inappropriate. In my understanding, **shrink** and **compress** should be accomplished before communication. If we add the **shrink** here, it will be done after the feature fusion at the ego agent side. I'm not sure if it's appropriate. Please help me to determine this part. Also, I suggest to use single-scale version.

Reply:
Yeah, I can understand your concern.  What if we shrink first and then fuse them?

Reply:
Sure. That's reasonable.